.github/workflows/check-tooling.yml

```diff
@@ -35,6 +35,9 @@ jobs:
 
       - run: yarn nx list @fluentui/workspace-plugin
 
+      # pre-build react-jsx-runtime - as generate-api executor doesn't provide copy assets capability
```

Comment:
https://github.com/microsoft/fluentui/issues/33200

Reply:
Would it be feasible and logical to include this as a dependency for `gererate-api` in nx.json, or perhaps even integrate it into an executor?

Reply:
yeah

my initial thinking when building these executors was to not make generate-api part of `build` rather invoke it via nx task dependency definition

`"build": "dependsOn": ["^build","generate-api"]`

after some thinking I opted out from doing that as `build` in our scope ( library domain ) needs integrated both generate-api (.d.ts) and build (js transpilation) - to make things encapsulated while composable right.

Now, this issue is rather exotic in context of react-jsx-runtime project, so while I think it's easy and doable to decouple copy assets into a separate executor/function and add it to both build and generate-api, it creates a bit verbose configuration options that start to be misleading IMO. 

Only purpose of generate-api should be run `tsc` and rollup types , not completely "sold" in incorporating assets api because this 1 use case we have is a good idea ATM.

Definitely open to more brainstorming here 💪, ty